src/main/java/net/rcarz/jiraclient/tempo/Team.java

```diff
@@ -176,6 +187,32 @@ public static boolean delete(RestClient restClient, Integer id) throws JiraExcep
 
     }
 
+    public static List<User> getTeamMembers(RestClient restClient, Integer id) throws JiraException {
```

Comment:
Let's not make it static - just method of the Team object
Feel free to take a look on User and groups field/methods